ivy/functional/frontends/torch/pointwise_ops.py

```diff
@@ -286,6 +286,7 @@ def clamp(input, min=None, max=None, *, out=None):
 clip = clamp
 
 
+@with_unsupported_dtypes({"2.0.1 and below": ("bfloat16", "uint16")}, "torch")
 @to_ivy_arrays_and_back
 def mul(input, other, *, out=None):
     input, other = torch_frontend.promote_types_of_torch_inputs(input, other)
```

Comment:
In this one, I get a similar `KeyError: 'cpu'` in the paddle backend, which needs to be resolved. Usually these errors are due to an incorrect framework version being referenced in `@with_unsupported_dtypes`.

Reply:
In the rest of the backends I see `nested_map` errors for `bfloat16`. These errors appear in other frontend function tests as well and they are probably caused by a bug in the testing pipeline. 
Failing dtypes should only be excluded from the test (i.e. added to `@with_unsupported_dtypes`) if the original function does not support them. Otherwise modifying the decorator is not the correct approach here.